src/Controller/CRUDController.php

```diff
@@ -429,6 +429,10 @@ public function batchAction()
             $action = $request->request->getAlnum('action');
             $idx = $request->request->get('idx', []);
             $allElements = $request->request->getBoolean('all_elements');
+
+            $request->request->set('idx', $idx);
```

Comment:
Manipulating the original request seems a little bit tricky. I'd consider this solution as a workaround.
IMO, without any test or comment covering this decision, it will be easy to miss these lines in future commits.

Reply:
Maybe it's better to not manipulating the request and just add the values to the data array?

```php
$idx = $request->request->get('idx', []);
$allElements = $request->request->getBoolean('all_elements');

$data = array_merge($request->request->all(), [
    'idx' => $idx,
    'all_elements' => $allElements,
]);
```

Reply:
But the request is passed to this finalAction, maybe this breaks a final action that's uses these values.

Reply:
I didn't analyzed the flow, but if the manipulated request needs to be passed to another method, maybe we should verify why is that behavior required, and in the worst case, clone the original request, set the required values in the request's bags and finally pass it to the `finalAction()`.